test/integration/openssl.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
@mattclay I'd like to have this file there because I'd like to be able to test those modules outside the `ansible-test` scope.

The idea is to test those modules with different OpenSSL implementation:

  * OpenSSL
  * LibreSSL
  * BoringSSL